controllers/provider-openstack/charts/internal/cloud-provider-config/values.yaml

```diff
@@ -0,0 +1,14 @@
+kubernetesVersion: version
+#[Global]
+authUrl: https://keystone/v3/
+domainName: %q
```

Comment:
These should contain concrete values, either reasonable defaults or example values.